apps/etterlatte-saksbehandling-ui/client/src/components/person/saksoversikt.tsx

```diff
@@ -6,59 +7,108 @@ import { Heading, Tag } from '@navikt/ds-react'
+  const [lastet, setLastet] = useState<boolean>(false)
+  const [behandlinglisteError, setBehandlinglisteError] = useState<IBehandlingsammendrag[]>()
+  const [grunnlagshendelserError, setGrunnlagshendelserError] = useState<Grunnlagsendringshendelse[]>()
+
   const behandlinger = behandlingliste ? behandlingliste : []
   const sakId = behandlinger[0]?.sak
 
   const sortertListe = behandlinger.sort((a, b) =>
     new Date(b.behandlingOpprettet!) > new Date(a.behandlingOpprettet!) ? 1 : -1
   )
 
+  useEffect(() => {
+    const getBehandlingsListeAsync = async (fnr: string) => {
+      const response = await hentBehandlingerForPerson(fnr)
+
+      if (response.status === 'ok') {
+        setBehandlingliste(response?.data)
+      } else {
+        setBehandlinglisteError(response?.error)
+      }
+
+      setLastet(true)
+    }
+
+    const getGrunnlagshendelserAsync = async (fnr: string) => {
+      const response = await hentGrunnlagsendringshendelserForPerson(fnr)
+
+      if (response.status === 'ok') {
+        setGrunnlagshendelser(response?.data.hendelser)
+      } else {
+        setGrunnlagshendelserError(response?.error)
+      }
+
+      setLastet(true)
+    }
+
+    if (fnr) {
+      getBehandlingsListeAsync(fnr)
+      getGrunnlagshendelserAsync(fnr)
+    }
+  }, [])
+
+  if (
+    (behandlinglisteError && behandlingliste === undefined) ||
+    (grunnlagshendelserError && grunnlagshendelserError === undefined)
+  ) {
+    return (
+      <Container>
+        {behandlinglisteError && <div>{JSON.stringify(behandlinglisteError)}</div>}
+        {grunnlagshendelserError && <div>{JSON.stringify(grunnlagshendelserError)}</div>}
+      </Container>
+    )
+  }
+
   return (
     <>
-      <SaksoversiktWrapper>
-        <HeadingWrapper>
-          <Heading spacing size="xlarge" level="1">
-            Barnepensjon
-          </Heading>
-          <div className="details">
-            <Tag variant={tagColors[INasjonalitetsType.NASJONAL]}>
-              {formaterEnumTilLesbarString(INasjonalitetsType.NASJONAL)}
-            </Tag>
-          </div>
-        </HeadingWrapper>
-        <ToKolonner>
-          {{
-            left: (
-              <>
-                {sakId !== undefined ? (
-                  <EkstraHandlinger>
-                    <ManueltOpphoerModal sakId={sakId} />
-                  </EkstraHandlinger>
-                ) : null}
-                <div className="behandlinger">
-                  <h2>Behandlinger</h2>
-                  <Saksliste behandlinger={sortertListe} />
-                </div>
-              </>
-            ),
-            right: grunnlagshendelser?.length ? (
-              <HendelseBorder>
-                <Grunnlagshendelser hendelser={grunnlagshendelser} />
-              </HendelseBorder>
-            ) : null,
-          }}
-        </ToKolonner>
-      </SaksoversiktWrapper>
+      <Spinner visible={!lastet} label={'Laster'} />
+      {lastet && (
```

Comment:
Ved `useApiCall`-funksjonen så får vi out of the box en lastet-variabel per api-call, så slipper vi å ha denne som sier true, selv om bare en av de er lastet

Reply:
https://jira.adeo.no/browse/EY-1534